### DISCUSSION TITLE
Mouse eye physiology?

### DISCUSSION THREAD

**Discussion by torwag** [sn-lab/MouseGoggles#11] [Q&A] [answered]
Hi,
I find that project interesting.  Just wondering how much did you consider the differences between the human eye and a mouse eye.
I am not an expert, but a quick search tells me that mouse eyes are substantial different related to the the field of depth,  colour recognition, spatial resolution, numerical aperture, frames per second,  sensitivity, etc.

Using an display made to trick the human eye into seeing movement,  might not work as well for mouses?!

Just wanted to point that out, so that it can be out of the way if research is done with it.  A reviewer of a journal would ask that question.

**Comment by misaacson01** [comment, marked as answer]
Hi @torwag, thank you for the question! Yes this is 100% made with mouse eye physiology in mind. As is, this would not be a good headset for humans -- it's too small, too low resolution, and it would be blurry for anyone who's even slightly near-sighted (like I am). It's even a bit too small and low res for rats! But I do believe it's well-made for small mouse eyes, which have [relatively poor acuity](https://ohsu.pure.elsevier.com/en/publications/independent-visual-threshold-measurements-in-the-two-eyes-of-free-2) and could be called [far-sighted](https://digitalcommons.wayne.edu/cgi/viewcontent.cgi?article=1748&context=oa_dissertations). On top of the theory, we're working on some behavioral tests that will hopefully be convincing to reviewers that mice are seeing meaningful things ;)

**Comment by BENETNATH** [comment]
Hi,
really interesting device.
Do you have real use video to see how the mouse behaves on the device ?

**Comment by misaacson01** [comment]
@BENETNATH We do have a lot of recorded video, it's on our to-do list to edit some so you can see what the mouse is seeing. We'll upload some examples eventually, keep an eye out for them!

**Comment by axolotlWorks** [comment]
What is the recommended lens-pupil distance?

**Comment by misaacson01** [comment]
@axolotlWorks Short answer: as close as you can get it to the mouse's eye
Longer answer: We're still working on fully characterizing the optics, but it's set up to be relatively invariant to the exact eye position. That said, if the eye is too far away or off center it will distort/blur/darken the image and reduce the FOV, particularly on the edges, so ideally you want the eye to be close to and centered on the lens

Just a note: we're working on some design updates over the next few weeks so bear in mind the files in the repo are still in prototype stage. If you do end up building one soon, keep us posted on how it goes, we'd love some feedback!

**Comment by ranczlab** [comment]
I had a discussion with Frank Schaeffel some years back when considering building a similar device. My main concern was close-focusing distance and what "contact lens" should be used. Perhaps you'll find Frank's opinion useful. 

This is from Frank:
"It is intresting that nobody can tell the true (functional) refractive state of the mouse. Most of the optical techniques agree that the refractive state is between +5 and +15 D but due to the fact that the light reflected from the retina may come from more proximal layers (closer to the vitreous) than the photoreceptor layer, it is likely that the mouse is more myopic than that. If all light would come from the retina-vitreous interface, it would even be about 35 D more myopic than what is optically measured. I am attaching an interesting paper on this issue. Schmucker and me (Vision Res 2006) found that mice see best at 30 cm distance without any lenses. This would argue that they are not very myopic at least.
The depth of focus issue is probably clear - there is optical depth of focus, in this case just because the optical aberrations appear to be high, and neural depth of focus, limited by the spatial frequencies that are detected by the retina etc. Both cause a depth of focus of about 10 D in both directions for the range of Sfs that mice can see (max 0.5 cyc/deg). The best contrast sensitivity in behavioral studies was found at 0.1 cyc/deg (Buse et al, J Neurosci 31, 11351-11361, 2011).
At the end, I would take the 30 cm distance (3D focus) and the 10 D depth of focus into account. This will be 13 D, which is about 7 cm viewing distance. But I could imagine that the image goes seriously out of focus when the targets are much closer than that. This is just a guess, however."

And here is a relevant paper:
[Optical properties of the mouse eye](https://pubmed.ncbi.nlm.nih.gov/21483598/)

**Comment by misaacson01** [comment]
@ranczlab Thanks for the comment, those references are great! I wasn't familiar with that Geng et al 2011 paper that suggests mouse eyes are more myopic, that's interesting and a little confusing to me. I was more familiar with research like that Schaeffel paper which seems to suggest far objects are in focus. This kind of behavioral research is more relevant to our purposes as well since our end goal is not to produce the sharpest image on the retina (definitely don't need or even want to be better than reality) but to elicit meaningful behaviors.

As for our current headset design, comparing it to the 2006 Schaeffel paper that also used lenses, I think our headset would be equivalent to a 30 cm distance display with a +3.3 D lens - putting it nicely between +0-7 D which produced the best results (unless I made some simple mistake somewhere). The lens powers tested in that paper are pretty huge, but a +7 D lens and a 30 cm display producing good result... that suggests hyperopia right?

Before we made our current headset, when we were modeling different optical configurations, we were originally planning slightly closer focal distances and more complicated lens arrays for wider FOV stimulation. But, since we set some further constraints on our system - namely, that it should be simple to build and easy to use - we have avoided optical configurations that need the eyes to be very accurately positioned on the lenses. And it seemed to us that infinity focus is good as we can get.
